engine/src/main/java/org/terasology/rendering/opengl/PostProcessor.java

```diff
@@ -211,8 +152,8 @@ public void saveScreenshot() {
             return;
         }
 
-        int width = buffers.sceneFinal.width();
-        int height = buffers.sceneFinal.height();
+        int width = buffersManager.getFBO("ocUndistorted").width();
+        int height = buffersManager.getFBO("sceneFinal").height();
```

Comment:
Thanks for finding this one. I've just fixed it. 

_Seems like a copy-paste issue 😭._